ecs-cli/modules/cli/compose/compose_app.go

```diff
@@ -142,11 +141,3 @@ func ProjectDown(p ecscompose.Project, c *cli.Context) {
 		log.Fatal(err)
 	}
 }
-
-// common function for all unsupported operations
-func unsupportedOperation(c *cli.Context) {
```

Comment:
Why is this removed?

Reply:
Because its not used at all. So I just removed it. `make build` and `make test` run fine without it.